lib/python/rose/gtk/dialog.py

```diff
@@ -262,7 +262,7 @@ def _process(cmd_args, stdout=sys.stdout, stderr=sys.stderr):
     for line in iter(proc.stderr.readline, ""):
         stderr.write(line)
     proc.wait()
-    stdout.read()  # Magically keep it alive.
+    stdout.read()  # Magically keep it alive!?
```

Comment:
Technically in feedback the intention was to convey that we should improve or remove this comment, but the humour is appreciated(!)

Reply:
The problem is, I don't really know (and/or do not yet want to find out) what we are trying to magically keep alive. It probably should have a *FIXME* label.

The statement before this is `proc.wait`, so in theory, the child process is already finished, so are we trying to keep something else alive?

Reply:
Fine, we will deal with this with future gui work.